ontology/termit-glosář.ttl

```diff
@@ -70,9 +70,9 @@ termit-pojem:plný-uživatel-termitu
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 termit:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
-                "TermIt full user"@en , "Plný uživatel TermItu"@cs ;
+                "Editor"@en , "Editor"@cs ;
         <http://www.w3.org/2004/02/skos/core#scopeNote>
-                "User with full access to TermIt."@en , "Uživatel s plným přístupem k systému TermIt."@cs .
+                "She can view, create, edit and delete vocabularies, terms and resources. She can also comment on terms. She cannot manage users."@en , "Může prohlížet, vytvářet, měnit a mazat slovníky, pojmy a zdroje. Pojmy může rovněž komentovat. Nemůže spravovat uživatele."@cs .
```

Comment:
Why only she? Thats racist :)

Reply:
AFAIK it is suggested to use **they** instead of **he/she**. At least I read it in instructions for paper authors in the Semantic Web journal.

Reply:
May be (I do not think that TermIt must follow SWJ instructions), Magic: the Gathering use He or She :) They is probably better

Reply:
During my past career in paper writing I already went through all the mentioned options - she / he or she / they . I personally don't care - happy to change to **He or she**, e.g.

Reply:
> Why only she? Thats racist :)

Considering women a 'race' is racist itself :-)